### DISCUSSION TITLE
Usage stats of my repositories

### DISCUSSION THREAD

**Discussion by zengyh1900** [community/community#37097] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to know how many individual users of my repositories are. 
Stars and contributors can not reflect the usage stats directly. 

I think the stats can come from the number of individual git clones. 
How can I get it?
This feature is essential for me to check which repository has the most users.

**Comment by FatChicken277** [comment, marked as answer]
You can see that metric in Insights->Traffic

![image](https://user-images.githubusercontent.com/60363061/197559425-8ffc5f78-7f3c-4fec-bbe4-762adfdbae97.png)

If you want more information about metrics, visit the [documentation](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository)

If I answered your question, then you should close the question by click accept answer button :heart:

**Reply by zengyh1900** [reply]
> You can see that metric in Insights->Traffic
> 
> ![image](https://user-images.githubusercontent.com/60363061/197559425-8ffc5f78-7f3c-4fec-bbe4-762adfdbae97.png)
> 
> If you want more information about metrics, visit the [documentation](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository)
> 
> If I answered your question, then you should close the question by click accept answer button ❤️

Hi @FatChicken277 
Thank you for your reply! 
However, it seems that I can only see unique cloners in recent 2 weeks. 
Can I get the number of unique cloners for a longer period?

**Reply by FatChicken277** [reply]
I understand, unfortunately it is not possible, since it is not implemented by github.

As an alternative you could use the [github api](https://docs.github.com/en/rest/metrics/traffic#get-repository-clones) to compile the information every x time or you can use this [github action](https://github.com/marketplace/actions/github-repo-stats) that allows you to store your statistics and create a +14 days accumulated report that you can export as html or pdf, even create a dedicated page from github pages, example: https://jgehrcke.github.io/ghrs-test/jgehrcke/covid-19-germany-gae/latest-report/report.html

Limitation: You have to create this flow from the moment you create the repository because metrics older than 14 days cannot be obtained

**Reply by zengyh1900** [reply]
Thanks! @FatChicken277

**Reply by antonkomarev** [reply]
@zengyh1900 if you want to collect data for a longer period you could try to use free service [Ÿ HŸPE](https://yhype.me)
